### PR TITLE
fix(Datatable): set rows action pane width to auto

### PR DESCRIPTION
**Pull request by julesbarr**

### CODE REVIEW

File: src/components/_internal/BaseDropdownMenu/DropdownMenu.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ export const DropdownLink = styled(Padbox).withConfig({
   shouldForwardProp: (prop) => !FORWARD_PROP_EXCEPTIONS.includes(String(prop)),
 })<DropdownLinkProps>`
   width: 100%;
+  min-width: 140px;
```
Comment (ajkl2533) [inline src/components/_internal/BaseDropdownMenu/DropdownMenu.tsx:33]: Should be in rem

File: src/components/_internal/BaseTable/columns/actionsColumn.tsx [hunk 2]
```diff
@@ -50,15 +50,19 @@ export const actionsColumn = {
...
       ...action,
       onClick: () => action.onClick(row.id, row.original),
     }))(rowActions);
 
     return (
       <Inline justify="center" stretch="start">
-        <DropdownMenu actions={actions} paneWidth={140} placement="bottom-end">
+        <DropdownMenu
+          actions={actions}
+          paneWidth={rowActionsPaneWidth}
```
Comment (ajkl2533) [inline src/components/_internal/BaseTable/columns/actionsColumn.tsx:63]: This seems to be overly complicated. I was thinking about setting the `paneWidth="auto"` and letting the content decide on how wide the menu should be. It should also limit the number of errors in tests :)

Comment (julesbarr) [reply src/components/_internal/BaseTable/columns/actionsColumn.tsx:63]: that's the default value, 'auto' and it's auto-adjusting based on the content

File: src/components/_internal/BaseDropdownMenu/DropdownMenu.tsx (hunk 1 above)
Comment (julesbarr) [reply src/components/_internal/BaseDropdownMenu/DropdownMenu.tsx]: 👍

File: src/components/_internal/BaseTable/columns/actionsColumn.tsx (hunk 2 above)
Comment (ajkl2533) [reply src/components/_internal/BaseTable/columns/actionsColumn.tsx:63]: Yeah I know I don't want to pollute Datatable component with another property if we can do the sizing automatically. I think that DT is already too complicated

**Comment by ajkl2533** [comment]
@julesbarr could you please change the commit message from `fix` to `feat`

**Review by ajkl2533 — APPROVED** [review]

**Pull request merged**